src/api/spec/features/webui/users/user_admin_edit_spec.rb

```diff
@@ -1,12 +1,14 @@
 require 'browser_helper'
 
-RSpec.describe "User's admin edit page", :js do
+RSpec.describe "User's admin edit page", :js, :vcr do
```

Comment:
Why does this need vcr now?

Reply:
To mock a backend query when the user removes their Admin permissions and gets redirected to the root_path. The user is still logged in.

Reply:
But admin permissions or redirects have nothing to do with the backend no?

Reply:
> But admin permissions or redirects have nothing to do with the backend no?

Now that the redirection is fixed, yes.

Before, when an admin user removed their admin permissions from the users page, it got repeatedly redirected to the users page. You need admin permissions to access the users configuration page.

After the changes, when an admin user removed their admin permissions from the users page, it gets redirected to the main page. The redirection loop is fixed. In the feature specs, a cassette mocking the backend queries of the main page for that user is needed. That's basically what is described in the commit body.

Reply:
I removed the cassette. The test fails now. It seems we cannot visti the main page without mocking some queries to the backend...

Reply:
If you only remove the cassette but not the `:vcr ` from the spec yes...

Reply:
It seems it doesn't make a difference. Same failures...